packages/apps/webpack.config.js

```diff
@@ -156,6 +156,7 @@ function createWebpack ({ alias = {}, context, name = 'index' }) {
       new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
       new webpack.DefinePlugin({
         'process.env': {
+          DOUGHNUT_SUPPORT: JSON.stringify(true),
```

Comment:
should set it to `false` by default or alternatively disable it on production build i.e use `isProd`

Reply:
done.